nucliadb_utils/nucliadb_utils/transaction.py

```diff
@@ -117,7 +120,11 @@ def received(waiting_for: WaitFor, event: Event, raw_data: bytes):
             pb = Notification()
             pb.ParseFromString(data)
             if pb.uuid == waiting_for.uuid and pb.action == action_type:
-                if waiting_for.seq is None or pb.seqid == waiting_for.seq:
+                if (
+                    waiting_for.seq is None
+                    or pb.seqid == waiting_for.seq
+                    or action_type != Notification.Action.INDEXED
```

Comment:
I don't understand the reason for this or condition. Can you explain?

Reply:
If we're waiting on the resource to be indexed, the seq id will not match.